README.md

```diff
@@ -40,13 +40,12 @@ out an open PR:
 
 ### Snapshot Builds
 
-For the daring, snapshot builds are available. These builds are created nightly and have undergone no formal QA, so they should never be run in production.
+For the daring, snapshot builds are available. These builds are created nightly and have undergone no formal QA, so they should never be run in production. All builds are 64 bit.
 
 | platform |  |
 | --- | --- |
 | OSX | [tar](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-darwin-x86_64.tar.gz) |
-| Linux x64 | [tar](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-linux-x86_64.tar.gz) [deb](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-amd64.deb) [rpm](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-x86_64.rpm) |
-| Linux x86 | [tar](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-linux-x86.tar.gz) [deb](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-i386.deb) [rpm](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-i686.rpm) |
+| Linux | [tar](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-linux-x86_64.tar.gz) [deb](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-amd64.deb) [rpm](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-x86_64.rpm) |
 | Windows | [zip](https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha2-SNAPSHOT-windows-x86.zip) |
```

Comment:
This should be updated to reference the 64 bit change `kibana-6.0.0-alpha2-SNAPSHOT-windows-x86_64.zip`